File: taskManager/labfinal_210042111/IssueRepository.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace labfinal_210042111
{
    public class IssueRepository
    {

        private Dictionary<int, Issue> repository = new Dictionary<int, Issue>();

        public IssueRepository(int id, Issue i)
        {
            repository.Add(id, i);
           
        }
        public void addIssue(Issue issue)
        {
            repository[issue.Id] = issue;
        }

           
        public void removeIssue(int id)
        {
            repository.Remove(id);
        }

        public Issue getIssue(int id)
        {
            return repository.ContainsKey(id) ? repository[id] : null;

        }

        public bool Contains(int id)=>repository.ContainsKey(id);
        public IEnumerable<Issue> GetAll() => repository.Values;


    }
}

File: TaskManager/210042141_TaskManager/src/IssueRepository.java
import java.util.HashMap;
import java.util.Map;

public class IssueRepository {
    private Map<Integer, Issue> issueMap = new HashMap<>();

    public void addIssue(Issue issue){
        issueMap.put(issue.getId(), issue);
    }

    public Issue getIssue(int id){
        return issueMap.get(id);
    }

    public void removeIssue(int id){
        issueMap.remove(id);
    }
}
